Platform-Linux/1-Ansible/ansible-jumpbox-connection.md

```diff
@@ -0,0 +1,40 @@
+There are multiple methods. The method I will be describing will be a method using the inventory file.
```

Comment:
I may be missing it but a document header (single `#`) with a title may be appropriate.

We also discussed having author tags on new documents, so they would be something like:

```
# some title
**Author**: someone
---
```

Or something, sometimes the `---` make the text between the headers the same size which is iffy.